tests/rkt_socket_activation_test.go

```diff
@@ -86,9 +86,14 @@ func TestSocketActivation(t *testing.T) {
 
 	rnd := r.Int()
 
+	// Write unit files directly to /run/systemd/system because systemd-v219 does
```

Comment:
This comment is a bit confusing. I would write something like:
```
Write unit files directly to runtime system units directory (/run/systemd/system) to avoid calling LinkUnitFiles - it is buggy in systemd v219 as it does not work with absolute paths.
```